go-controller/pkg/ovn/ovn.go

```diff
@@ -60,6 +60,9 @@ type namespaceInfo struct {
 	// the policy itself.
 	networkPolicies map[string]*namespacePolicy
 
+	//the UUID of the namespaces wide port group that contains all the pods in the namespace
```

Comment:
space after the "//". And "namespace-wide" not "namespaces wide"

Reply:
fixed